src/coupons/coupons.service.ts

```diff
@@ -22,14 +22,19 @@ export class CouponsService {
   }
 
   async getAllCoupons() {
-    const docs = await this.couponModel.find().exec();
+    const docs = await this.couponModel
+      .find()
+      .populate('required_menus')
+      .exec();
     return docs;
   }
 
   async getCouponById(id: string) {
     return await this.couponModel
       .findById(id)
-      .orFail(new NotFoundException('Coupon not found'));
+      .populate('required_menus')
+      .orFail(new NotFoundException('Coupon not found'))
+      .exec();
```

Comment:
Please create a DTO for these services as well (with required_menus populated)

Reply:
Fixed [9d4b4f9](https://github.com/meeorder/meeorder-backend/pull/139/commits/9d4b4f9bf2d0135d706c7935cebf28ef024c715c). Does it correct?